### PR TITLE
refactor metadata module pom file

### PR DESCRIPTION
**Pull request by Zzm0809**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by zackyoungh — APPROVED** [review]
LGTM

File: dinky-metadata/dinky-metadata-hive/pom.xml [hunk 1]
```diff
@@ -56,14 +56,36 @@
...
-            <id>cloudera</id>
-            <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
-        </repository>
-    </repositories>
+    <!--    <repositories>-->
+    <!--        <repository>-->
+    <!--            <id>cloudera</id>-->
+    <!--            <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>-->
+    <!--        </repository>-->
+    <!--    </repositories>-->
```
Comment (leechor) [inline dinky-metadata/dinky-metadata-hive/pom.xml:95]: redundency

Comment (Zzm0809) [reply dinky-metadata/dinky-metadata-hive/pom.xml:95]: > redundency

If it is the cdh version of hive, you need to open the comments here

**Review by aiwenmo — APPROVED** [review]
LGTM

**Review by leechor — APPROVED** [review]
LGTM

**Pull request merged**